lib/microform/test_methods.rb

```diff
@@ -0,0 +1,39 @@
+require "minitest/stub_any_instance"
+require "minitest/mock"
+
+module Microform
+  module TestMethods
+    def assert_submits form_kind, stub: nil, &block
+      controller = self.class.name.gsub(/Test\z/, "").constantize
+      form = form_kind.new OpenStruct.new(foo: :bar)
+
+      __microform_form_kind = form_kind
```

Comment:
The lambda that get defined and passed to `stub_any_instance` ends up being executed inside of the controller instance that is under test, so naming the variables without mangling the name (I think) introduces the possibility that the variable name clashes with a method name in the controller. I can add a note about this as a comment.